assets/src/components/block-mover/block-drag-area.js

```diff
@@ -25,13 +14,13 @@ export const IconDragHandle = ( { isVisible, className, icon, onDragStart, onDra
 			{
 				( { onDraggableStart, onDraggableEnd } ) => (
 					<div
-						className={ dragHandleClassNames }
+						className={ className }
 						aria-hidden="true"
 						onDragStart={ onDraggableStart }
 						onDragEnd={ onDraggableEnd }
 						draggable
 					>
-						{ icon }
+						{ children }
```

Comment:
This `<BlockDragArea>` now wraps the block (children), instead of appearing alongside it. This allows dragging the entire block.